alda-mode.el

```diff
@@ -153,14 +219,23 @@ Argument END The end of the selection to append from."
   (interactive)
   (alda-history-append-region (line-beginning-position) (line-end-position)))
 
+(defun alda-eval-region (start end)
```

Comment:
This function's name dosen't imply it's related to the interpreter at all, can we name it:

`alda-inf-eval-region`

or similar?

Reply:
This function seems to be having trouble parsing a multiline statement like:
```
piano: c d e
	e f g
	c d e
```

I'm getting errors like:
```
p> p> [27713] ERROR Unexpected '	' at line 1, column 1.
p> p> [27713] ERROR Unexpected '	' at line 1, column 1.
p> [27713] ERROR Unexpected '	' at line 1, column 1.
p> 
```

to the alda repl. I'm not sure if this is a problem in the alda interpeter itself (it proably is). If it's a problem with the alda repl, I'll look into why it's doing that in my spare time.

Reply:
Hmm.. Actually this code might not even work in my own emacs, even using the non-repl implementation. every time I have to update the code to
```
piano: c d e
e f g
c d e
```